.changeset/config.json

```diff
@@ -7,10 +7,6 @@
   "access": "public",
   "baseBranch": "main",
   "updateInternalDependencies": "patch",
-  "ignore": [
-    "ensnode",
-    "@ensnode/ensrainbow",
-    "@docs/ensnode",
-    "@docs/ensrainbow"
-  ]
+  "prettier": false,
+  "ignore": ["ensnode", "@ensnode/ensrainbow", "@docs/ensnode", "@docs/ensrainbow"]
```

Comment:
Auto formatting applied by running `pnpm biome check --write .` in the root directory of that repo.